dpnp/dpnp_algo/dpnp_elementwise_common.py

```diff
@@ -43,9 +44,459 @@
     "DPNPReal",
     "DPNPRound",
     "DPNPUnaryFunc",
+    "ufunc",
+    "unary_ufunc",
+    "binary_ufunc",
 ]
 
 
+import dpctl.tensor._tensor_elementwise_impl as ti
+
+import dpnp.backend.extensions.vm._vm_impl as vmi
+
+
+class ufunc:
+    """
+    ufunc()
+
+    Functions that operate element by element on whole arrays.
+
+    Calling ufuncs
+    --------------
+    op(*x[, out], **kwargs)
+
+    Apply `op` to the arguments `*x` elementwise, broadcasting the arguments.
+
+    Parameters
+    ----------
+    *x : {dpnp.ndarray, usm_ndarray}
+        Input arrays.
+    out : {None, dpnp.ndarray, usm_ndarray}, optional
+        Output array to populate.
+        Array must have the correct shape and the expected data type.
+    order : {None, "C", "F", "A", "K"}, optional
+        Memory layout of the newly output array, Cannot be provided
+        together with `out`. Default: ``"K"``.
+    dtype : {None, dtype}, optional
+        If provided, the destination array will have this dtype. Cannot be
+        provided together with `out`. Default: ``None``.
+    casting : {"no", "equiv", "safe", "same_kind", "unsafe"}, optional
+        Controls what kind of data casting may occur. Cannot be provided
+        together with `out`. Default: ``"safe"``.
+
+    Limitations
+    -----------
+    Keyword arguments `where` and `subok` are supported with their default values.
+    Other keyword arguments is currently unsupported.
+    Otherwise ``NotImplementedError`` exception will be raised.
+
+    """
```

Comment:
due to some reason the whole description of `dpnp.ufunc.outer` was inserted to `dpnp.ufunc` page, but expected to have only summary per each method, like [here](https://numpy.org/doc/stable/reference/generated/numpy.ufunc.html#numpy.ufunc)